extensions/transform/org.eclipse.smarthome.transform.jsonpath/src/main/java/org/eclipse/smarthome/transform/jsonpath/internal/JSonPathTransformationService.java

```diff
@@ -65,7 +65,8 @@ public String transform(String jsonPathExpression, String source) throws Transfo
                 return transformationResult.toString();
             }
         } catch (PathNotFoundException e) {
-            throw new TransformationException("Invalid path '" + jsonPathExpression + "' in '" + source + "'");
+            logger.debug("path not found in JSON string (PathNotFoundException)");
```

Comment:
For all it's worth, I'd have stayed with the previous string.
Please start the line with a capital letter and don't mention details a user can not comprehend.
Debug level sems a good choice.